cli/dcoscli/util.py

```diff
@@ -23,7 +23,7 @@ def wrapper(*args, **kwargs):
         try:
             result = func(*args, **kwargs)
         except docopt.DocoptExit as e:
-            emitter.publish("Command not recognized\n")
+            emitter.publish("Invalid subcommand usage\n")
```

Comment:
Cool. It would be good to have a common string abstracted out for this, but we can leave this as future work.